plugins/@grouparoo/clickhouse/src/lib/table-import/getPropertyValues.ts

```diff
@@ -0,0 +1,142 @@
+import { validateQuery } from "@grouparoo/mysql/dist/lib/validateQuery";
```

Comment:
There have been some significant updates to this file in Postgres since we started this branch - please rebase and take a look.  There have also been additions to the tests - https://github.com/grouparoo/grouparoo/pull/2522 has the relevant changes

Reply:
Changes have been applied to `getPropertyValues`